connectors/es/sink.py

```diff
@@ -149,6 +150,9 @@ def _bulk_op(self, doc, operation=OP_INDEX):
 
     @tracer.start_as_current_span("_bulk API call", slow_log=1.0)
     async def _batch_bulk(self, operations, stats):
+        msg = "I think this will reproduce"
+        raise Exception(msg)
```

Comment:
I need to remove this, but this is how I've been testing my changes. This error throws every time `_batch_bulk` is called, and IMO it should crash the sync the _first_ time it's called.